content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Prometheus
 description: How to integrate with Prometheus.
-weight: 29
+weight: 30
```

Comment:
We often use weights increasing by 10 for all the docs in a folder (e.g., 10, 20, 30, ...). Then we can add new ones in between (e.g., 15) later, without changing the other docs. Maybe you want to do that for the integrations folder?

On the other hand, if you want them in alphabetical order, you could make them all the same weight. They will then appear in the alpha order of the doc names (in github), not the title, but you do have control of that too.